src/server/chromium.ts

```diff
@@ -122,7 +122,7 @@ export class Chromium implements BrowserType {
         // We try to gracefully close to prevent crash reporting and core dumps.
         // Note that it's fine to reuse the pipe transport, since
         // our connection ignores kBrowserCloseMessageId.
-        const t = transport || await platform.createWebSocketTransport(browserWSEndpoint!);
+        const t = transport || new platform.WebSocketTransport(browserWSEndpoint!);
         const message = { method: 'Browser.close', id: kBrowserCloseMessageId };
         t.send(JSON.stringify(message));
```

Comment:
Please do `return t.send(...)` or `await t.send(...)`; so that we catch any error during send and kill browser on promise reject.

Reply:
Done